src/freertos_drivers/sources

```diff
@@ -23,7 +23,7 @@ CXXSRCS += Fileio.cxx \
 
 
 ifeq ($(TARGET),freertos.mips4k.pic32mx)
-SUBDIRS += drivers_pic32mx795 plib_pic32mx795
+SUBDIRS += drivers_pic32mx530 plib_pic32mx530
```

Comment:
here it is enough to have all four listed, they will be independently compiled, each for the specific processor it targets -- see another comment.